.gitignore

```diff
@@ -149,3 +149,9 @@ tests/testing_data/CT_2D_head_moving.mha
 
 # profiling results
 *.prof
+
+tests/testing_data/MNI152_T1_2mm.nii.gz
```

Comment:
If this is something you're testing with it, and not one of our test files, it shouldn't be here I think.